### PR TITLE
Ports Bay's railguns, and assorted gun sprites

### PR DESCRIPTION
**Pull request by Anewbe**
- Coilgun is craftable, the rest are behind admin walls
- New sprites for the guns might be nice, but these will do for now.
- Grabbed a bunch of in-hand gun sprites while I was in the file

### CODE REVIEW

**Comment by FenodyreeAv** [comment]
Does unreliable 15 mean 15% to fail? And what happens on a fail?

**Comment by Anewbe** [comment]
Yes, and bad things I didn't look too hard into.

**Comment by Mechoid** [comment]
A failure causes a decent explosion centered on the wielder. Decent meaning probably gonna break a bone.

File: code/game/objects/items/weapons/RCD.dm [hunk 1]
```diff
@@ -164,6 +166,14 @@
 	w_class = ITEMSIZE_SMALL
 	origin_tech = list(TECH_MATERIAL = 2)
 	matter = list(DEFAULT_WALL_MATERIAL = 30000,"glass" = 15000)
+	var/remaining = 10
+
+/obj/item/weapon/rcd_ammo/large
+	name = "high-capacity matter cartridge"
+	desc = "Do not ingest."
```
Comment (SpadesNeil) [inline code/game/objects/items/weapons/RCD.dm]: ... What... what happens if you ingest it?

Comment (Anewbe) [reply code/game/objects/items/weapons/RCD.dm]: Nothing?

Comment (SpadesNeil) [reply code/game/objects/items/weapons/RCD.dm]: Well that's no fun. Why put a warning label if it's not dangerous?

**Comment by SpadesNeil** [comment]
seriously though

`merg dis`

**Comment by Woodratt** [comment]
Out of curiosity, pictures?

**Comment by Anewbe** [comment]
You are my bane, @woodratt. Will try to cobble together an image or two later.

**Comment by Anewbe** [comment]
railguns
Automatic, Flechette, Normal, Ghetto

**Comment by SpadesNeil** [comment]
[GUN FETISH INTENSIFIES]

File: code/game/objects/items/weapons/RCD.dm (hunk 1 above)
Comment (SpadesNeil) [reply code/game/objects/items/weapons/RCD.dm]: Remind me to make it possible to eat these, and also make them do horrible things.

**Pull request merged**